pkg/webui/styles/utilities/general.styl

```diff
@@ -43,11 +43,11 @@
       +media-query($width)
         for $num in (1..12)
           // Generate responsive grid item overrides.
-          .{$name}:item-{$num}
+          .{$name}\\:item-{$num}
```

Comment:
Seems like I was wrong about removing the backslashes. Also there's an issue with the linter about this. I'll take a look.